src/Gnugat/Medio/Exception/DomainException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Medio project.
+ *
+ * (c) Ivan Scherbak <dev@funivan.com>
```

Comment:
I'd like to be the only copyright owner of Medio, to avoid having to track down every contributor when a copyright changes is required (see: http://programmers.stackexchange.com/a/191965. I remember seeing a similar issue on an open source project, not sure which... Maybe doctrine?).

I value contributions, and I'd be glad to make you name visible in the README. Is it ok with you?